clouseau/patchanalysis.py

```diff
@@ -245,6 +257,21 @@ def attachmenthandler(attachments, bugid, data):
         'r-ed_patches': sum((a['is_patch'] == 1 or a['content_type'] == 'text/x-review-board-request') and sum(flag['name'] == 'review' and flag['status'] == '-' for flag in a['flags']) > 0 for a in bug['attachments']),
     }
 
+    # Store bug creator & assignee
+    assignee = bug.get('assigned_to_detail')
+    creator = bug.get('creator_detail')
+
+    # Search uplift requester & raw request from comment
```

Comment:
Can you move the uplift request part in the `uplift_info` function? We could merge the two functions in the future, for now let's keep them separated.